test/paper-dialog-scrollable.html

```diff
@@ -39,9 +35,9 @@
 
     <test-fixture id="basic">
       <template>
-        <section class="fixed-height vertical">
+        <section>
           <div class="header">Header</div>
-          <paper-dialog-scrollable>
+          <paper-dialog-scrollable class="fixed-height">
```

Comment:
This change looks like it will break any current usages where i've set the height on the section, and the p-d-s used to take the available height, no?